docs/en/quick-answers/troubleshooting/cant-use-file-linking-toolkit-app-chrome.md

```diff
@@ -73,7 +67,9 @@ If your computer is not connected to the Internet and can’t download the updat
 Under certain circumstances, Windows will report that a certificate has been imported successfully, but will not make it available to applications requiring it. This can be validated by visiting the certificate dialog on Windows. To access it, hit the Windows key and type **Internet Options**. On the **Internet Properties** dialog, switch to the **Content** tab and then click on the **Certificates** button. Finally, click on the **Trusted Root Certification Authorities** and look for **localhost**.
 
 ![Search Windows for internet options](images/windows-search-internet-options.png)
+
 ![Content tab of the internet properties](images/windows-internet-properties.png)
+
 ![Windows Certificates](images/windows-certificates.png)
 
 If the entry is missing, there is probably a group policy issue on your Windows domain or local computer. If it is present, we recommend you contact our [support team](support@shotgunsoftware.com).
```

Comment:
We should confirm that this link to an email address works. I suspect it should actually be a `mailto:` link, and I'm not sure our interpreter will deal with that. (Also I HATE mailto: links and think they should be abolished from the internet, so my personal suggestion would be to link here: https://support.shotgunsoftware.com/hc/requests/new)

Reply:
Gone with the requests new link as that is what we are doing everywhere else. good spot